quicktests/overlaying/tests/realistic/phones.js

```diff
@@ -0,0 +1,150 @@
+
+function makeData() {
+  "use strict";
+  var dataAustralia = [
+    {phoneType: "feature",  demographic: "Male",        value: 0.33},
+    {phoneType: "feature",  demographic: "Female",      value: 0.29},
+    {phoneType: "feature",  demographic: "Ages 16-24",  value: 0.19},
+    {phoneType: "feature",  demographic: "Ages 25-34",  value: 0.21},
+    {phoneType: "feature",  demographic: "Ages 35-44",  value: 0.28},
+    {phoneType: "feature",  demographic: "Ages 45-64",  value: 0.44},
+    {phoneType: "smart",    demographic: "Male",        value: 0.64},
+    {phoneType: "smart",    demographic: "Female",      value: 0.67},
+    {phoneType: "smart",    demographic: "Ages 16-24",  value: 0.77},
+    {phoneType: "smart",    demographic: "Ages 25-34",  value: 0.76},
+    {phoneType: "smart",    demographic: "Ages 35-44",  value: 0.69},
+    {phoneType: "smart",    demographic: "Ages 45-64",  value: 0.51}
+  ];
+
+  var dataIndia = [
+    {phoneType: "feature",  demographic: "Male",        value: 0.76},
+    {phoneType: "feature",  demographic: "Female",      value: 0.85},
+    {phoneType: "feature",  demographic: "Ages 16-24",  value: 0.70},
+    {phoneType: "feature",  demographic: "Ages 25-34",  value: 0.80},
+    {phoneType: "feature",  demographic: "Ages 35-44",  value: 0.86},
+    {phoneType: "feature",  demographic: "Ages 45-64",  value: 0.87},
+    {phoneType: "smart",    demographic: "Male",        value: 0.13},
+    {phoneType: "smart",    demographic: "Female",      value: 0.07},
+    {phoneType: "smart",    demographic: "Ages 16-24",  value: 0.14},
+    {phoneType: "smart",    demographic: "Ages 25-34",  value: 0.12},
+    {phoneType: "smart",    demographic: "Ages 35-44",  value: 0.09},
+    {phoneType: "smart",    demographic: "Ages 45-64",  value: 0.06}
+  ];
+
+  var dataRoK = [
+    {phoneType: "feature",  demographic: "Male",        value: 0.20},
+    {phoneType: "feature",  demographic: "Female",      value: 0.25},
+    {phoneType: "feature",  demographic: "Ages 16-24",  value: 0.08},
+    {phoneType: "feature",  demographic: "Ages 25-34",  value: 0.07},
+    {phoneType: "feature",  demographic: "Ages 35-44",  value: 0.21},
+    {phoneType: "feature",  demographic: "Ages 45-64",  value: 0.38},
+    {phoneType: "smart",    demographic: "Male",        value: 0.68},
+    {phoneType: "smart",    demographic: "Female",      value: 0.65},
+    {phoneType: "smart",    demographic: "Ages 16-24",  value: 0.86},
+    {phoneType: "smart",    demographic: "Ages 25-34",  value: 0.86},
+    {phoneType: "smart",    demographic: "Ages 35-44",  value: 0.66},
+    {phoneType: "smart",    demographic: "Ages 45-64",  value: 0.49}
+  ];
+
+
+  return [dataAustralia, dataIndia, dataRoK];
```

Comment:
It might make more sense to return an object instead of an array:
```Javascript
return {
  australia: dataAustralia,
  india: dataIndia,
  korea: dataRoK
}
```
Then later you can do
```Javascript
var ausDataset = new Plottable.Dataset(data.australia);
```
Which reads a bit nicer.